x-pack/plugins/reporting/export_types/csv_from_savedobject/types.d.ts

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
Almost everywhere in Reporting uses types files named `index.d.ts`.

My understanding is in those places, the `.d` is ideal because you can't have 2 files both named `index.d.ts`, but importing both works the same way.

For consistency, we should have Reporting type files also named `index.d.ts` -- but this seems to be a pre-existing problem.